### DISCUSSION TITLE
Lacking accuracy with videos of different length

### DISCUSSION THREAD

**Discussion by tompalmer** [theophanemayaud/video-simili-duplicate-cleaner#121] [General]
Hello, 

Compared to "Video comparer", I find that this software has troubles with videos where the length is dramatically different. 
What could be done about it ? What options in software can mitigate it ? Using less thumbnails ?

**Comment by theophanemayaud** [comment]
What do you mean dramatically different video lengths ? If two videos have different durations, they can’t be duplicates ! How would you want it to report two videos as a match if they’re not the approx same duration… if one is longer, it can’t be the same video ?!

**Reply by tompalmer** [reply]
Compilations and multi parts videos

Sometimes you will have a video in 5 parts, and without knowing it the full version somewhere in your hardrive. You will want to keep the fragmented version or the other. 

Video comparer has the ability to spot those as long as the video is up to 16 times longer than the other one (it's changeable)

**Reply by tompalmer** [reply]
extended cuts also ... lots of situations
